syntax = "proto3";


service CommunicationService{
 rpc  getFile(FileRequest) returns (stream FileChunk);
}
message FileChunk{
 bytes data=1;
 int64 chunk_number=2;
 int64 size=3;
}
message FileRequest {
  int64 startFrom=1;
  int64 keyl=2;
  int64 keyh=3;
  int64 fileNumber=4;
}
